### PR TITLE
Updated Query Service README

### PR DESCRIPTION
**Pull request by udasitharani**
Solves: #740

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -51,6 +59,14 @@ Need to update [https://github.com/SigNoz/signoz/tree/main/pkg/query-service](ht
 - comment out frontend service section at `docker/clickhouse-setup/docker-compose.yaml#L59`
 - comment out query-service section at `docker/clickhouse-setup/docker-compose.yaml#L38`
 - Install signoz locally without the frontend and query-service
+
+  - Add this to `query-service` section in the `docker-compose.yaml`:
```
Comment (makeavish) [inline CONTRIBUTING.md:63]: Please add the full path to `docker-compose.yaml`

**Review by makeavish — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (udasitharani) [reply CONTRIBUTING.md:63]: ah okay

Comment (makeavish) [reply CONTRIBUTING.md:63]: Very good catch.
Also one important step to start query-service locally was removed in this PR: https://github.com/SigNoz/signoz/commit/9c07ac376db8f0125b1fb8cadb129867af376e8e

Please add to line 72: `STORAGE=clickhouse ClickHouseUrl=tcp://localhost:9000 go run main.go`
Also add: Update path of `RELATIONAL_DATASOURCE_PATH` to `"./signoz.db"` at https://github.com/SigNoz/signoz/blob/05c79b71190decb76bee79e535ef3251090a0cd7/pkg/query-service/constants/constants.go#L33

Maybe we also need to expose clickhouse docker container port on 9000. 
Please try these steps once.

Comment (udasitharani) [reply CONTRIBUTING.md:63]: lemme try

**Comment by makeavish** [comment]
@udasitharani Looks like you have been busy, should I assign this issue to someone else?

**Comment by udasitharani** [comment]
I'm really sorry, should've notified. Please do.

**Comment by makeavish** [comment]
Closing this as this is fixed in https://github.com/SigNoz/signoz/pull/815

**Pull request closed without merging**